ai/views.py

```
@@ -0,0 +1,539 @@
+            else:
+                title = 'Untitled Chat'
+                
+        # If chat_id is provided, try to find existing chat
+        chat = None
+        is_new = False
+        if chat_id:
+            try:
+                chat = ChatSession.objects.get(id=chat_id, user=request.user)
+            except ChatSession.DoesNotExist:
+                # If chat doesn't exist, create a new one
+                pass
```

Ruff (0.8.2)

Use `contextlib.suppress(ChatSession.DoesNotExist)` instead of `try`-`except`-`pass`

(SIM105)